Wikipedia/Code/WMFExploreCollectionViewCell.m

```diff
@@ -4,26 +4,17 @@
 @implementation WMFExploreCollectionViewCell
 
 - (UICollectionViewLayoutAttributes *)preferredLayoutAttributesFittingAttributes:(UICollectionViewLayoutAttributes *)attributesToFit {
-    if ([attributesToFit isKindOfClass:[WMFCVLAttributes class]] && [(WMFCVLAttributes *)attributesToFit precalculated]) {
-        return attributesToFit;
```

Comment:
`precalculated` is set when the layout was calculated ahead of time and returned for the estimated layout. In theory, that should be the correct height so it doesn't need to be re-calculated here. Unfortunately in practice, it seems auto-layout gives two different height results.